app/src/main/java/com/willowtree/vocable/facetracking/FaceTrackFragment.kt

```diff
@@ -75,7 +74,7 @@ class FaceTrackFragment : ArFragment() {
         ReplaceWith("Nothing, this is what AR is expecting")
     )
     override fun onRequestPermissionsResult(requestCode: Int, permissions: Array<out String>, grantResults: IntArray) {
-        EasyPermissions.onRequestPermissionsResult(requestCode, permissions, grantResults, requireActivity())
+        requireActivity().onRequestPermissionsResult(requestCode, permissions, grantResults)
```

Comment:
I see this needs to be overridden because we're still extending from `ArFragment`, but can this be a no-op, since we're handling this via the ActivityResult stuff in MainActivity?